test/transcript/TranscriptEvent.test.ts

```diff
@@ -227,4 +227,40 @@ describe('TranscriptEvent', () => {
 
     expect(actualEvents.length).to.equal(0);
   });
+
+  it('handles one transcript event of transcript type with entities', async () => {
+    const event = makeSdkTranscriptWithEntities();
+    const data = SdkTranscriptFrame.encode(makeSdkTranscriptFrame([event])).finish();
+    logBase64FromUint8Array(data);
+    // The steps above generates data below passed directly as test input
+    const dataMessage = makeTranscriptDataMessageFrom(
+      TRANSCRIPT_EVENT_TEST_VECTORS.TRANSCRIPT_ENTITY
+    );
+    const actualEvents = TranscriptEventConverter.from(dataMessage);
+    expect(actualEvents.length).to.equal(1);
+    if (!(actualEvents[0] instanceof Transcript)) {
+      assert.fail();
+      return;
+    }
+    expect(actualEvents[0].results.length).to.eql(1);
```

Comment:
nit: I meant you can `expect()` + `eql()` (deep comparison) in one line like the following:

```
expect(actualEvents).to.eql([
  {
     results: [
       {
          alternatives: [
              {
                  items: [
                       { confidence: 1, stable: true },
                       { stable: false },
                  ]
              }
          ]
       }
     ]
  }
]);

```

This helps:
- We can remove Array.length check
- We know an exact shape expected for the transcription data message